src/HomeWorkApp.java

```diff
@@ -0,0 +1,129 @@
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        int a = 55;
+        int b = -40;
+        int value = -88;
+        String stringToPrint = "Строка на печать.";
+        int amountToPrint = 5;
+        int year = 1100;
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+        System.out.println(checkRange(a, b));
+        checkValue(value);
+        System.out.println(checkIfValueIsNegative(value));
+        printString(stringToPrint, amountToPrint);
+        System.out.println(isLeapYear(year));
+
+    }
+
+    /**
+     * 1. Создайте метод printThreeWords(),
+     * который при вызове должен отпечатать в столбец три слова: Orange, Banana, Apple.
+     */
+    public static void printThreeWords() {
+        System.out.println("""
+                Orange
+                Banana
+                Apple""");
+    }
+
+    /**
+     * 2. Создайте метод checkSumSign(), в теле которого объявите две int переменные a и b,
+     * и инициализируйте их любыми значениями, которыми захотите.
+     * Далее метод должен просуммировать эти переменные, и если их сумма больше или равна 0,
+     * то вывести в консоль сообщение “Сумма положительная”,
+     * в противном случае - “Сумма отрицательная”;
+     */
+    public static void checkSumSign() {
+        int a = 10;
+        int b = -50;
+        if (a + b >= 0) {
+            System.out.println("Сумма положительная");
+        } else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+
+    /**
+     * 3. Создайте метод printColor() в теле которого задайте int переменную value и инициализируйте ее любым значением.
+     * Если value меньше 0 (0 включительно), то в консоль метод должен вывести сообщение “Красный”,
+     * если лежит в пределах от 0 (0 исключительно) до 100 (100 включительно),
+     * то “Желтый”, если больше 100 (100 исключительно) - “Зеленый”;
+     */
+    public static void printColor() {
+        int value = -44;
+        if (value <= 0) {
+            System.out.println("Красный");
+        } else if (0 < value && value <= 100) {
```

Comment:
Эта проверка уже выполнена выше. Не стоит усложнять
```suggestion
        } else if (value <= 100) {
```